redi-connect-front/src/components/organisms/Footer.tsx

```diff
@@ -18,7 +18,7 @@ const Footer = () => {
               <p>&copy; {year} By ReDI School</p>
             </Columns.Column>
             <Columns.Column size={4} className="is-hidden-tablet">
```

Comment:
`className="is-hidden-tablet"` is equal to `responsive={{` tablet: { hide: { value: true } } }}`

Technically the second one should be used. However I still fine it ugly. We should however agree and proceed with one syntax everywhere.